hafta1/hafta1.md

```diff
@@ -609,11 +609,29 @@ Bu yüzden lütfen dışarıdan bir uygulama kuracaksınız, forumlara ya da you
 
 ---
 
+## AUR(Arch User Repository) Nedir ?
+
+AUR, archlinux topluluğunun geliştirdikleri uygulama paketlerini, kütüphaneleri ve daha bir çok şeyi tüm archlinux topluluğuyla paylaştığı repodur. AUR da bulunan paketler ana repoda yoktur. AUR da bulunan paketler topluluk tarafından oylanarak olumlu yönde oy almış paketler ana repoya taşınabilir.
```

Comment:
Herhangi bir insanın AUR reposuna bir şey ekleyebileceğini, Archlinux tarafından resmi olarak desteklenmediğini, AUR'dan kurulacak paketlerin güvenliğin kullanıcının sorumlu olduğundan da bahsedelim.

Bir de `AUR da` değil de `AUR'da` yazar mısın